server/src/main/java/com/hedera/block/server/config/BlockNodeConfigExtension.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.server.config;
+
+import com.google.auto.service.AutoService;
+import com.swirlds.common.config.BasicCommonConfig;
+import com.swirlds.common.metrics.config.MetricsConfig;
+import com.swirlds.common.metrics.platform.prometheus.PrometheusConfig;
+import com.swirlds.config.api.ConfigurationExtension;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Set;
+
+/** Registers configuration types for the server. */
+@AutoService(ConfigurationExtension.class)
```

Comment:
Is this required by the platform SDK config API, or did we chose to add this?
If it's not required, we shouldn't use the Google Autoservice (and if it is required, we might want to ask the Platform SDK to _consider_ removing that requirement).

Reply:
Yes it is indeed required by the Platform SDK.

I will open up a ticket for them to reconsider that change.